src/js/fiContainer.jsx

```diff
@@ -0,0 +1,59 @@
+var React = require('react');
+var divisionHeader = require('./divisionHeader.jsx');
+var fiStatus = require('./fiStatus.jsx');
+
+var fiContainer = React.createClass({
+
+  getInitialState: function(){
+    return this.updateDivisions(this.props.institutions);
+  },
+
+  updateDivisions: function(institutions){
+    var state = {
+      divisions: [
+        {text: 'Not Started', institutions: []},
+        {text: 'In Progress', institutions: []},
+        {text: 'Completed', institutions: []}
+      ]
+    };
+
+    institutions.forEach(function(institution){
+      var status = institution.status;
+      var index = 2;
+
+      if(status < 5) index = 1;
+      if(status === 0) index = 0;
+
+      state.divisions[index].institutions.push(institution);
+
+    });
+
+    return state;
+  },
+
+  updateInstitutions: function(institutions){
+    this.setState(this.updateDivisions(institutions));
+  },
+
+  render: function() {
+    return (
+      <div id="fiContainer">
+        {this.state.divisions.map(function(division, i){
+          var header = null;
+          if(division.institutions.length) header = React.createElement(divisionHeader, {text: division.text});
+          return (
+            <div key={i} className="divisionWrapper">
+              {header}
+              {division.institutions.map(function(institution, i){
+                return React.createElement(fiStatus, {key: i, institution: institution});
```

Comment:
I thought not, yet I was experiencing bugs when I had custom components declared with JSX (bizarrely, because it is certainly supposed). React.createElement removed the trouble so I stuck with that mode.